package.json

```diff
@@ -11,7 +11,8 @@
     "rollback": "scripts/rollback",
     "latest": "scripts/latest",
     "seed": "scripts/seed",
-    "lint": "scripts/lint"
+    "lint": "scripts/lint",
+    "knex": "knex"
```

Comment:
I'm actually not sure that I need this. I was just following the steps in the Learn 'Knex Migrations' article. It doesn't make a lot of sense to me why you would need to do that but here is a screenshot of that step in Learn 
![image](https://user-images.githubusercontent.com/24230076/28886654-1caec46e-776f-11e7-81bd-e668b8844f0c.png)